src/coresyf_image_crop/tests/test_image_crop.py

```diff
@@ -54,3 +53,7 @@ def test_apply_buffer_to_polygon(self):
                                                       crs_buffer)
 
         self.assertEqual(polygon_with_buffer.GetEnvelope(), expected_envelope)
+
+    def test_get_raster_resolution(self):
+            resolution = get_raster_resolution(self.test_image)
+            self.assertGreater(abs(resolution), 0)
```

Comment:
Correct this line to:
'self.assertEqual(resolution, 180)'

Reply:
Done.